cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -2001,4 +2001,13 @@ public static final class MetadataConsumer {
     public static final String METADATA_CONSUMER_EXTENSIONS_ENABLED_LIST = "metadata.consumer.extensions.enabled.list";
     public static final String METADATA_CONSUMER_EXTENSIONS_DIR = "metadata.consumer.extensions.dir";
   }
+
+  /**
+   * Constants for Source Control Management
+   */
+  public static final class SourceControl {
```

Comment:
nit, will `SourceControlManagement` be more practical?

Reply:
Changed.